languages/cpp/config.json

```
@@ -9,11 +9,17 @@
   },
   "exercises": {
     "concept": [
+      {
+        "slug": "basics",
+        "uuid": "dc8bb97c-5dfa-430f-ba77-3f50276debb8",
+        "concepts": ["basics"],
+        "prerequisites": [""]
```

This should be an empty array

```suggestion
        "prerequisites": []
```